doc/circulation/actions.md

```diff
@@ -72,17 +64,9 @@
 1. :100:__CHECKIN_4__: item in_transit (IN_TRANSIT_FOR_PICKUP)
 	1. :white_check_mark: :white_check_mark: :white_check_mark: __CHECKIN_4_1__: pickup location = transaction library →  (delivery_receive current loan, item is: at_desk(ITEM_AT_DESK))
 	1. __CHECKIN_4_2__: pickup location != transaction library →  (no action, item is: in_transit (IN_TRANSIT_FOR_PICKUP)) *Future: would be useful to track that an intermediary checkin has been made*
-1. :100:__CHECKIN_5__: item in_transit (IN_TRANSIT_TO_HOUSE)
-	1. __CHECKIN_5_1__: PENDING loan does not exist
-		1. :white_check_mark: :white_check_mark: :white_check_mark: __CHECKIN_5_1_1__: item location = transaction library →  (house_receive current loan, item is: on_shelf)
-		1. :white_check_mark: __CHECKIN_5_1_2__: item location != transaction library →  (no action, item is: in_transit (IN_TRANSIT_TO_HOUSE)) *Future: would be useful to track that an intermediary checkin has been made*
-	1. __CHECKIN_5_2__: PENDING loan exists
-		1. __CHECKIN_5_2_1__: pickup location of first PENDING loan = transaction library
-			1. __CHECKIN_5_2_1_1__: pickup location of first PENDING loan = item library →  (house_receive current loan, item is: at_desk(ITEM_AT_DESK))[automatic validate first PENDING loan]
-			1. __CHECKIN_5_2_1_2__: pickup location of first PENDING loan != item library →  (cancel current loan, item is: at_desk(ITEM_AT_DESK))[automatic validate first PENDING loan] *It is the case if the first PENDING loan has been deleted or unpriorised since last checkin*
-		1. __CHECKIN_5_2_2__: pickup location of first PENDING loan != transaction library
-			1. :white_check_mark: __CHECKIN_5_2_2_1__: pickup location of first PENDING loan = item library →  (no action, item is: in_transit (IN_TRANSIT_TO_HOUSE)) *Future: would be useful to track that an intermediary checkin has been made*
-			1. __CHECKIN_5_2_2_2__: pickup location of first PENDING loan != item library →  (checkin current loan, item is: in_transit (IN_TRANSIT_FOR_PICKUP))[automatic cancel current loan, automatic validate first PENDING loan] *It is the case if the first PENDING loan has been deleted or unpriorised since last checkin*
+1. __CHECKIN_5__: item in_transit (IN_TRANSIT_TO_HOUSE)
```

Comment:
I do not think you need to remove the `IN_TRANSIT_TO_HOUSE` here. This is the main method that actually do the job for you. So it better to have it detailed here. Maybe you can have two texts:
- the details one to explain how it is coded
- the new one is actually for the librarian

Reply:
I thought that your code doesn't take into account the case of items `IN_TRANSIT_TO_HOUSE` with pending loans, since it is a case we want to avoid (see my previous response).

Reply:
I think we need to talk about that. Because on my side even if specification doesn't talk about pending loan, the system can be in a state where pending loan exist during a IN_TRANSIT_xx exists and someone make a "checkout" of the item. It generates problems that I didn't resolve yet.